README.md

```diff
@@ -8,6 +8,7 @@
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 [![PyPI version](https://badge.fury.io/py/atoMEC.svg)](https://badge.fury.io/py/atoMEC)
 [![Python 3.8](https://img.shields.io/badge/python-3.8-blue.svg)](https://www.python.org/downloads/release/python-380/)
+[![codecov](https://codecov.io/gh/atomec-project/atoMEC/branch/develop/graph/badge.svg?token=V66CJJ3KPI)](https://codecov.io/gh/atomec-project/atoMEC)
```

Comment:
Is the  `token=.....` actually needed? I guess as soon as the PR is merged, the badge dor the `develop` branch will work as it does currently for the `codecov` branch: 

- https://codecov.io/gh/atomec-project/atoMEC/branch/codecov/graph/badge.svg → ![](https://codecov.io/gh/atomec-project/atoMEC/branch/codecov/graph/badge.svg)
- https://codecov.io/gh/atomec-project/atoMEC/branch/develop/graph/badge.svg → ![](https://codecov.io/gh/atomec-project/atoMEC/branch/develop/graph/badge.svg)

Reply:
If they provide the link with this token, its fine. In scikitlearn, a token is also attached, see https://raw.githubusercontent.com/scikit-learn/scikit-learn/main/README.rst . I think it doesn't really matter. OK, you can merge it!